### PR TITLE
api: Add stable marshaling

### PR DESCRIPTION
**Pull request by cthulhu-rider**
source code is ready and ready for review. Some tests left

### CODE REVIEW

File: api/status/codes.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package status
+
+// All supported status codes.
+const (
+	OK                        = 0
```
Comment (carpawell) [inline api/status/codes.go:5]: have you thought about some helper bit operations here? just to describe why it is 1024, 1025, 1026, 1027 and then 2048

File: api/accounting/encoding.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/nspcc-dev/neofs-sdk-go/internal/proto"
+)
+
+const (
+	_ = iota
+	fieldDecimalValue
+	fieldDecimalPrecision
+)
+
+// MarshaledSize returns size of the Decimal in Protocol Buffers V3 format in
```
Comment (carpawell) [inline api/accounting/encoding.go:13]: is it handmade? there was some concept with code generation, have you considered it? tree things were made with it: https://github.com/nspcc-dev/neofs-node/blob/22259ce755199d6e87331a3b58cc14449a36f69e/pkg/services/tree/types_neofs.pb.go

File: api/status/codes.go (hunk 1 above)
Comment (cthulhu-rider) [reply api/status/codes.go:5]: this is not needed, code is just a code, nobody cares what number is it

File: api/accounting/encoding.go (hunk 2 above)
Comment (cthulhu-rider) [reply api/accounting/encoding.go:13]: > have you considered it?

not yet, we need a proper testing 1st (it's coming with this PR). Then we can experiment with codegen

**Review by roman-khimov — APPROVED** [review]

File: api/accounting/encoding.go (hunk 2 above)
Comment (carpawell) [reply api/accounting/encoding.go]: imo, a generator should have been written first, that is an idea about it. but it is late

**Review by carpawell — APPROVED** [review]

**Pull request merged**